autoload/sy/repo.vim

```diff
@@ -317,6 +322,52 @@ function! s:preview_hunk(_sy, vcs, diff) abort
   noautocmd call feedkeys("\<c-w>p", 'nt')
 endfunction
 
+" Function: #revert_hunk {{{1
+function! sy#repo#revert_hunk() abort
+  if exists('b:sy') && !empty(b:sy.updated_by)
+    call sy#repo#get_diff(b:sy.updated_by, function('s:revert_hunk'))
+  endif
+endfunction
+
+function! s:revert_hunk(_sy, vcs, diff) abort
+  call sy#verbose('s:revert_hunk()', a:vcs)
+
+  let [header, hunk] = s:extract_current_hunk(a:diff)
+  if empty(hunk)
+    return
+  endif
+
+  let [_old_line, new_line, _old_count, _new_count] = sy#sign#parse_hunk(header)
+
+  for line in hunk
+    echom new_line
```

Comment:
I'll remove all the `:echom` debug messages for now. If you want to keep them, we can add them back later using `sy#verbose()`, then those would only be shown when the command is executed like this: `:verbose SignifyHunkUndo`.